code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/builders/ManualCarouselNotificationBuilder.kt

```diff
@@ -136,7 +136,7 @@ internal object ManualCarouselNotificationBuilder {
      * @param items the list of [CarouselPushTemplate.CarouselItem] objects to be displayed in the filmstrip carousel
      * @return a list of `CarouselPushTemplate.CarouselItem` objects that were successfully downloaded
      */
-    private fun downloadCarouselItems(
+    internal fun downloadCarouselItems(
```

Comment:
Can you mark these functions that you made internal for testing with `@VisibleForTesting` ?

Reply:
I guess no @spoorthipujariadobe, getting the below error upon doing that
<img width="781" alt="Screenshot 2024-06-11 at 4 27 49 PM" src="https://github.com/adobe/aepsdk-ui-android/assets/169874009/f1818e8a-9a8c-4f72-bd08-b31a22eea28e">